lib/do_profile/do_create.js

```diff
@@ -354,6 +354,22 @@ function _createProfile(opts, cb) {
                 return;
             }
 
+            var chosenKey = ctx.keyChoices.find(function (keyChoice) {
+                return (data.keyId === keyChoice.keyId);
+            });
+
+            if (chosenKey.pubKey.type === 'ed25519') {
+                console.log(common.ansiStylizeTty('\n\n# Skipping Docker ' +
+                    'Setup\n', 'bold'));
+                console.log(wrap80('You have selected an ED25519 key. ' +
+                    'ED25519 keys can not be used for generating Docker ' +
+                    'client certificates until EDDSA is adopted by the ' +
+                    'CA/Browser Forum. Use an RSA or ECDSA key to generate ' +
+                    'a Docker client certificate.\n'));
```

Comment:
This should mention something like:

> Use `--keyId` to select an ECDSA or RSA for signing your Docker certificate.

Also, we should make sure that works :-) I think it does, but we should also make sure it does.

Reply:
Just confirmed an ECDSA keypair works with Docker here too.